src/frontend/handlers.go

```diff
@@ -52,14 +53,28 @@ var (
 			"renderCurrencyLogo": renderCurrencyLogo,
 		}).ParseGlob("templates/*.html"))
 	plat platformDetails
+	rdb  = redis.NewClient(&redis.Options{
+		Addr:     "localhost:6379", // Cambia esto si Redis está en otra dirección o puerto
+		Password: "",               // Agrega la contraseña si tu Redis la requiere
+		DB:       0,                // Base de datos a usar (0 por defecto)
+	})
 )
 
+func init() {
+	_, err := rdb.Ping(context.Background()).Result()
+	if err != nil {
+		panic(fmt.Sprintf("No se pudo conectar a Redis: %v", err))
```

Comment:
Maybe consider using error handling with logging instead of panic to gracefully degrade functionality. Otherwise application may crash if the error occurs.